pages/user/[username]/index.tsx

```diff
@@ -106,38 +106,25 @@ Contributor.PageLayout = ProfileLayout;
 export default Contributor;
 
 export const getServerSideProps = async (context: UserSSRPropsContext) => {
-  return await handleUserSSR(context);
-};
-
-export type UserSSRPropsContext = GetServerSidePropsContext<{ username: string }>;
-
-export async function handleUserSSR({ params }: GetServerSidePropsContext<{ username: string }>) {
-  const { username } = params!;
-
-  async function fetchUserData() {
-    try {
-      const req = await fetch(`${process.env.NEXT_PUBLIC_API_URL}/users/${username}`, {
-        headers: {
-          accept: "application/json",
-        },
-      });
-
-      if (!req.ok) {
-        return null;
-      }
+  const { username } = context.params!;
+  const req = await fetch(`${process.env.NEXT_PUBLIC_API_URL}/users/${username}`, {
```

Comment:
you can try the following for the server-side request forgery warning. The `username` should be validated as well to avoid something like `username = `../user/delete`.

```suggestion
  const req = await fetch(`${new URL("/users/${username}", process.env.NEXT_PUBLIC_API_URL})`, {
```

Reply:
There's also the URL param validators functions I wrote in: https://github.com/open-sauced/app/pull/3467

```suggestion
  if (!isValidUrlSlug(username)) {
    throw new Error("Invalid input");
  }
  
  const req = await fetch(`${process.env.NEXT_PUBLIC_API_URL}/users/${username}`, {
```

which should in theory always error if an invalid slug (like trying to escape a URL with `;`) is provided

Reply:
Looks like it's not complaining about the request forgery anymore 👍🏼 😎 👍🏼